app/furniture/marketplace/carts_controller.rb

```diff
@@ -0,0 +1,24 @@
+class Marketplace
+  class CartsController < Controller
+    def update
+      authorize(cart)
+      cart.update(cart_params)
+      respond_to do |format|
+        format.html do
+          redirect_to cart.room.location
+        end
+        format.turbo_stream do
+          render turbo_stream: [turbo_stream.replace(dom_id(cart), cart)]
```

Comment:
Is this logic using turbo stream to AJAX-like update/rerender only the cart item elements on the webpage instead of rerendering the whole page?

Reply:
Yep!